threadedServer.py

```diff
@@ -28,12 +28,25 @@
 # Doesn't really matter what's in here, just make sure only 2 for now
 # Otherwise have to change the logic of the server
 player_data = [
-    Player(0, 0, 50, 50, "Red", 100),
-    Player(50, 50, 60, 60, "Green", 95)
+    "Hello server data",        # P1 data to be sent to P2
```

Comment:
This is dummy data, and doesn't really matter. Probably gonna do something with this init data to help with ease of starting the clients.